pom.xml

```diff
@@ -404,24 +404,24 @@
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-compiler-plugin</artifactId>
-          <version>3.1</version>
+          <version>3.7.0</version>
           <configuration>
-            <source>1.7</source>
-            <target>1.7</target>
+            <source>1.8</source>
+            <target>1.8</target>
           </configuration>
         </plugin>
         <plugin>
           <groupId>org.apache.felix</groupId>
           <artifactId>maven-bundle-plugin</artifactId>
-          <version>2.3.7</version>
+          <version>3.3.0</version>
           <extensions>true</extensions>
           <configuration>
             <instructions>
               <Embed-Dependency>*;inline=false;scope=compile</Embed-Dependency>
               <Embed-Transitive>true</Embed-Transitive>
               <Embed-Directory>lib</Embed-Directory>
               <!--Only @Plugin classes in the export packages will be included as plugin-->
-              <_exportcontents>co.cask.kudu.*,org.apache.kudu.*</_exportcontents>
+              <_exportcontents>io.cdap.plugin.kudu.*,org.apache.kudu.*</_exportcontents>
```

Comment:
doesn't this have to be io.cdap.plugin.kudu without .*? Because is specifies packages, not classes?